### DISCUSSION TITLE
Increase suffering % by actually suffering?

### DISCUSSION THREAD

**Discussion by sojuenjoyer** [Aizistral-Studios/Enigmatic-Legacy#363] [Ideas]
spending 99.5% of the time on a server with the ring is almost impossible if you dont do it within the first few days. As playing with a modpack makes you skim over it till later.
What if time spent burning, getting down to low health, dying, starving, or a custom end game item can increase the % required or lower the min requirement to use 99.5% items?

**Comment by Aizistral** [comment]
Maybe someday, not soon.
